CMakeLists.txt

```diff
@@ -92,8 +92,7 @@ else()
   target_link_libraries(rmm INTERFACE CUDA::cudart)
 endif(CUDA_STATIC_RUNTIME)
 
-target_link_libraries(rmm INTERFACE rmm::Thrust)
-target_link_libraries(rmm INTERFACE spdlog::spdlog_header_only)
+target_link_libraries(rmm INTERFACE rmm::Thrust spdlog::spdlog_header_only ${CMAKE_DL_LIBS})
```

Comment:
What is `CMAKE_DL_LIBS` and where is it defined?

Reply:
@germasch suggested it [here](https://github.com/rapidsai/rmm/pull/575#discussion_r498568695) as a way to pull in dl, which is needed for the stack traces. I wonder if this will cause issue for other platforms though.